wrappers/kn/src/nativeMain/kotlin/zxingcpp/Barcode.kt

```diff
@@ -0,0 +1,137 @@
+package zxingcpp
```

Comment:
And one more thing that I missed so far: please add a comment header with copyright and license information. See e.g. [here](https://github.com/zxing-cpp/zxing-cpp/blob/6faf0f8286c3259834c80f37e9b950bdb85ecd17/wrappers/android/zxingcpp/src/main/cpp/ZXingCpp.cpp#L1). That should be added to all `*.kt` files.

Reply:
Definitely the first one. And regarding the Copyright, that is up to you. Note that I deliberately refrained from making changes to your PR directly (which would have saved you from a number of trivial changes that I was requesting) to not muddle the authorship info in the git-log after squashing the PR. There might be some argument to be made that both our names should show up in a Copyright line each. ;)

Reply:
Ahh, thank you for your dedicated efforts to support my contributions :)

Reply:
copyrights are added: https://github.com/zxing-cpp/zxing-cpp/pull/719/commits/55ca66f3609551f4e8bf3e9731e47aaacf3c3b0e